configs/cli/config.toml

```diff
@@ -0,0 +1,26 @@
+[channel]
+  state = ""
+  status = ""
+  topic = ""
+
+[filter]
+  contact = ""
+  email = ""
+  limit = ""
+  metadata = ""
+  name = ""
+  offset = ""
```

Comment:
`build/mainflux-cli config name "thing_1"` for this command getting error 
```error
error: error converting filter to Uint64 : strconv.ParseUint: parsing "true": invalid syntax
```

Reply:
![image](https://github.com/mainflux/mainflux/assets/43080232/5bfdac88-ca7e-4199-923d-565d70c047f8)